### PR TITLE
feat(invoice_display_name_plan): Add invoice display name to plan

### PR DESCRIPTION
**Pull request by ivannovosad**
## Context

Add the ability to overwrite plan’s and subscription’s names in invoices.

## Description

This PR handles only plans and subscriptions invoice display names, charges, fees will be implemented in a follow-up PR.

### CODE REVIEW

File: spec/factories/plans.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@
   factory :plan do
     organization
     name { Faker::TvShows::SiliconValley.app }
+    invoice_display_name { Faker::TvShows::BreakingBad.episode }
```
Comment (vincent-pochet) [inline spec/factories/plans.rb:7]: Good choice ;)

**Review by vincent-pochet — APPROVED** [review]
:rocket:

File: app/graphql/mutations/plans/create.rb [hunk 2]
```diff
@@ -15,6 +15,7 @@ class Create < BaseMutation
       argument :code, String, required: true
       argument :description, String, required: false
       argument :interval, Types::Plans::IntervalEnum, required: true
+      argument :invoice_display_name, String, required: false
```
Comment (jdenquin) [inline app/graphql/mutations/plans/create.rb:18]: GraphQL/ExtractInputType: Consider moving arguments to a new input type

File: app/graphql/mutations/plans/update.rb [hunk 3]
```diff
@@ -15,6 +15,7 @@ class Update < BaseMutation
       argument :description, String, required: false
       argument :id, String, required: true
       argument :interval, Types::Plans::IntervalEnum, required: true
+      argument :invoice_display_name, String, required: false
```
Comment (jdenquin) [inline app/graphql/mutations/plans/update.rb:18]: GraphQL/ExtractInputType: Consider moving arguments to a new input type

**Pull request merged**